StressTest.ps1

```diff
@@ -0,0 +1,24 @@
+﻿[CmdletBinding()]
+param(
+    [string]
+    $BinSkimFolder = "bld\bin\x64_Release\netcoreapp3.1",
+
+    [string]
+    $OutputFolder = "c:\temp\",
```

Comment:
>$OutputFolder [](http://example.com/codeflow?start=4&length=13)

shouldn't we consider creating a special directory in temp for this data?

Reply:
Created a scripts directory.
we should also move the other files in the future.